Components/Queues/QueueMessage.php

```diff
@@ -72,6 +77,11 @@ public function setPersistent($persistent)
         }
     }
 
+    public function setMessageId(int $messageId)
```

Comment:
To make this `int` typehint work, we need to define in the class `declare(strict_types=1);`

Reply:
Done.